docs/README.md

```diff
@@ -0,0 +1,48 @@
+# Fortuna Documentation
+
+The documentation for Fortuna can be found [here](https://aws-fortuna.readthedocs.io/en/latest/).
+
+## Build the documentation
+
+The documentation for Fortuna is comprised of a series of notebooks. To serve these locally, there are two steps that need to be taken that we outline below.
+
+### Prerequisites
+
+To build the documentation, first install Fortuna and it's dependencies by following the [installation instructions](https://github.com/awslabs/fortuna#installation). Next, install the documentation requirements through
+```bash
+poetry install -E docs
+```
+Finally, install `jupytext` through either pip or conda, details can be found [here](https://github.com/mwouts/jupytext#install). 
```

Comment:
How come does this need to be a separate step? Could `jupytext` be included in `docs`?

Reply:
Excellent point. Done!